src/shared/bus-get-properties.c

```diff
@@ -3,6 +3,7 @@
 #include "bus-get-properties.h"
 #include "bus-message-util.h"
 #include "rlimit-util.h"
+#include "set.h"
```

Comment:
Interestingly(?) rlimit-util.h -> strv.h -> hashmap.h -> set.h .
Also, you added set.h in bus-message-util.h. So, not necessary.